governator-core/src/main/java/com/netflix/governator/InjectorBuilder.java

```diff
@@ -62,6 +63,23 @@ public static InjectorBuilder fromModules(List<Module> modules) {
         return new InjectorBuilder(Modules.combine(modules));
     }
     
+    public static <T> InjectorBuilder bindInstance(final Object o) {
```

Comment:
We probably want to avoid this since we're trying to stay true to the Guice API.  We can discuss this further but either way let's keep this separate from this PR.